setup.py

```diff
@@ -5,51 +5,51 @@
 
 try:
     import Cython
+
     pass
 except ImportError:
-    print "*** cannot import Cython"
+    print("*** cannot import Cython")
     pass
 
-if 'Cython' in globals():
-    print "*** info: Building from Cython"
+if "Cython" in globals():
+    print("*** info: Building from Cython")
     ext_files = ["bloomfilter/_bloomfilter.pyx"]
 else:
-    print "*** info: Building from C"
+    print("*** info: Building from C")
     ext_files = ["bloomfilter/_bloomfilter.c"]
 
 ext_modules = [
-    Extension("bloomfilter._bloomfilter", ext_files, include_dirs=['bloomfilter'])
+    Extension(
+        "bloomfilter._bloomfilter", ext_files, include_dirs=["bloomfilter"]
+    )
 ]
 
 requirements = []
 if sys.version_info[0] < 3 and sys.version_info[1] < 7:
-    requirements.append('importlib')
+    requirements.append("importlib")
 
-setup(name           = 'bloomfilter',
-    version          = '0.1.0',
-    description      = 'Bloom Filter (Python)',
-    license          = 'MIT License',
-    url              = 'http://github.com/seomoz/bloomfilter-py',
-    author           = 'Big Data',
-    author_email     = 'vadim@moz.com',
-    packages         = ['bloomfilter'],
-    package_dir      = {'bloomfilter': 'bloomfilter'},
-    install_requires = [requirements],
-    tests_require    = [
-        'coverage',
-        'nose',
-        'pylint',
-    ],
-    scripts = [],
+setup(
+    name="bloomfilter",
+    version="0.2.0",
+    description="Bloom Filter (Python)",
+    license="MIT License",
+    url="http://github.com/seomoz/bloomfilter-py",
+    author="Big Data",
+    author_email="vadim@moz.com",
```

Comment:
Should we consider changing this to a group email?  (perhaps whatever the current pro services team email is?)

Reply:
... that *is* a bad email, what should we replace it with ?

Reply:
Checking with @seomozSLE

Reply:
Updated.